test/check_event_log.txt

```diff
@@ -0,0 +1,18 @@
+; Checks the pattern of event log file. Makes sure the data rows
+; Corresponding to each layer have the correct format.
+; Checks the shader hashes across different rows and guarantees they are
+; consistent.
+; Counts the number of memory and frame time logs and check if they are
+; as expected.
+; CHECK-DAG: compile_time_layer_init,{{[0-9]+}}
+; CHECK-DAG: memory_usage_layer_init,{{[0-9]+}}
+; CHECK-DAG: frame_time_layer_init,{{[0-9]+}}
+; CHECK: create_shader_module_ns,{{[0-9]+}},[[SHADER1:0x[a-zA-Z0-9]+]],{{[0-9]+}}
+; CHECK-NEXT: create_shader_module_ns,{{[0-9]+}},[[SHADER2:0x[a-zA-Z0-9]+]],{{[0-9]+}}
+; CHECK: shader_module_first_use_slack_ns,{{[0-9]+}},[[SHADER1]],{{[0-9]+}}
+; CHECK-NEXT: shader_module_first_use_slack_ns,{{[0-9]+}},[[SHADER2]],{{[0-9]+}}
+; CHECK: create_graphics_pipelines,{{[0-9]+}},"[[[SHADER1]],[[SHADER2]]]",{{[0-9]+}}
+; CHECK: memory_usage_present,{{[0-9]+}},{{[0-9]+}},{{[0-9]+}}
+; CHECK-COUNT-99: {{(memory_usage_present|frame_present),[0-9]+,[0-9]+,[0-9]+[[:space:]].*(frame_present|memory_usage_present),[0-9]+,[0-9]+,[0-9]+}}
```

Comment:
The two other test already checks the exact number of frames and frame presents. I think that here it would be sufficient to check that there at least one each